docs/5g-on-helium/5g-poc.mdx

```diff
@@ -119,22 +117,155 @@ must:
   - Be registered with the SAS (Outdoor Radios Only).
 - Broadcast at least 1 "heartbeat" per Reward Period.
 
-### Genesis Phase 3: Speed Test and Quality of Service
+### Genesis Phase 3: Speed Tests and Quality of Service
 
-> - 5G Hotspots perform periodic speed tests which will be considered in reward calculations.
+> - Reward Period is approximately 24 hours, with Reward submission occurring at approximately 01:30
+>   UTC.
+> - 5G Hotspots perform periodic Speed Tests which will be considered in reward calculations.
 > - A minimum Download speed, Upload speed, and Latency will be required by each 5G Hotspot to
 >   ensure it is providing quality coverage.
 
-A 5G Hotspot will perform a speed test every 12 hours with the results being averaged over the
-previous 6 tests.
-
-To meet the speed test requirements as outlined in
+To meet the Speed Test requirements as outlined in
 [HIP-53](https://github.com/helium/HIP/blob/main/0053-mobile-dao.md#oracle-operations), a 5G Hotspot
 must have a rolling average Download speed of at least 100Mbps **AND** a rolling average Upload
 speed of at least 10Mbps **AND** a rolling average Latency of 50ms or less.
 
-**A 5G Hotspot that does not meet all three of these requirements will be excluded from rewards
-calculations in the given Reward Period.**
+A 5G Hotspot will perform a Speed Test approximately every 12 hours with the results being averaged
+by the MOBILE PoC Oracle over the previous `6` tests. This moving average is reset following a speed
+test gap of `>12 hours` and a minimum of `2` Speed Test results are required to be eligible for
+Genesis Rewards.
+
+:::note Moving Average
+
+The "Moving Average", also known as a "Rolling Average" is the average of the 6 most recent speed
+test results for each download, upload, and latency.
```

Comment:
The "Moving Average", also known as a "Rolling Average" is the average of at least 2 most recent speed test results for each download, upload, and latency. 6 is the max number of speed tests used in the moving average calculation. This means that test that came before 6th the most recent test, won't be considered in the calculation.

Minimum 2 and maximum 6 speed tests are required for moving average calculation.

Reply:
how about:

```markdown
The "Moving Average", also known as a "Rolling Average" is the average of at least 2 but no more
than 6 most recent speed test results for each download, upload, and latency.

This means that when more than 6 Speed Test results are available, results that came before 6th most
recent test won't be considered in the calculation.
```

Reply:
Perfect!